rfc/2019-12-11-Talent-Plan-Courses.md

```diff
@@ -0,0 +1,35 @@
+# Proposal: Talent Plan Courses
+
+- Author(s): [Ling Jin](https://github.com/JinLingChristoher)
+- Last updated: 2020-01-05
+- Discussion at: [Incubating Program: Talent Plan Courses](https://github.com/pingcap/community/issues/130)
+
+## Background
+
+Talent Plan is a study project developed by PingCAP, which aims to help people get in touch with TiDB and TiKV. At the moment, only two study roadmaps were provided, and the content is not well-designed. So we propose Talent Plan Courses, a new version of the study project that contains more topics and elaborate contents.
+
+## Course Framework
+
+![course map](../media/rfc-talent-plan-courses.png)
+
+The picture above is the design of Talent Plan Courses which is made up of multiple topics. Each topic contains multiple courses and has a study goal. In detail, for open source collaboration, we hope participants acquired basic knowledge about open source community operation and knows the basic usage of related tools. For Infrastructure systems, multiple secondary topics will be contained, programming assignments and related reading materials will be provided. Furthermore, Programming languages will be provided as crash study materials for people without related experience, after learning the series of courses, participants should have a good knowledge and skills in infrastructure systems or other related fields. Deep dive into the TiDB ecosystem also is an indispensable part to help people get to know more about TiDB related projects.
+
+As the development of courses, more topics and the detailed course contents would be appended in the future.
+
+## Rationale
+
+There are many open study resources online. Our courses can borrow ideas from world-class universities and MOOCs as well. The courses will be practice-oriented, with well-designed programming assignments as the core. Reading materials are also included.
```

Comment:
I don't think we have the right to "borrow ideas from world-class universities and MOOCs". What we are doing is gathering the open learning materials together to make the learning path easier and providing some assignments to help students check their learning outcomes.